src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "System.Runtime.InteropServices.RuntimeInformation" :  "4.0.0-beta-*",
     "System.ServiceModel.Duplex": "4.0.1-beta-*",
```

Comment:
No, it's not there. We'd have to conditionally compile - and this project doesn't do any checking for FEATURE_CORECLR or the like. We'd also run into the same issue we had when dealing with System.Net.DnsInformation in that the ToF runtime would have a problem distinguishing when to include the reference and when not to. 

Best to find a way to avoid this entirely...